### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
In this branch me and lucas created a modal window when you click on "see project"

### CODE REVIEW

**Comment by AdnanOlarmmi** [comment]
Hi @Feliperto 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
You have submitted a project, that you created the project by yourself without involving your partner, so please can you make your partner commit to the project 😃 as this is pair programming activity.
The only evidence to show that Lucas did the project with you is for him to commit, kindly encourage him to do so, thanks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by laiifuu — COMMENTED** [review]
# STATUS: INVALID :stop_sign: 

## Hi team :wave: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where linters check is red. (Stylelint to be specific)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -270,6 +275,137 @@ body.scroll .navbar__section {
...
+  height: 92vh;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  border: 1px solid var(--borderColor);
+  border-radius: 16px;
+  padding: 16px;
+  background: #fff;
+  z-index: 1;
+}
```
Comment (laiifuu) [inline css/style.css]: When opening the popup in both mobile and desktop view, the content overflows its container, and so the bottom content will not appear as part of the container: 
- Mobile view:   
    127 0 0 1_5500_(iPhone SE) (2)
- Desktop view: 
    

Adding `overflow: scroll;` should fix this and make the popup scrollable.

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

## Hi team :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
- [x] Manipulation of the popup with JS :heavy_check_mark: 
- [x] Nice scroll animation :+1:  

## Required Changes ♻️
- [ ] The curve of the About Me section looks funny in mobile view: 
    127 0 0 1_5500_(iPhone SE) (1)
- [ ] The images on the popups in desktop view lose their clarity and look weirdly stretched, kindly fix this issue. 
example: 
    image

_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great job on the funtionality for the pop up💯
- [ ] Followed GitHub flow 🔥
- [ ] No Linters error ✅
- [ ] Good job working on the design 🔥⭐

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**